lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,7 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <img src="/static/images/xseries-certificate-visual.png" alt="XSeries Program Certificates" aria-hidden="true">
```

Comment:
If you set ```alt=""``` you don't need the ```aria-hidden="true"```. It is a quicker way of achieving the same thing.